hc/front/tests/test_my_checks.py

```diff
@@ -37,7 +37,7 @@ def test_it_shows_red_check(self):
         self.check.save()
 
         self.client.login(username="alice@example.org", password="password")
-        r = self.client.get("/checks/")
+        r = self.client.get("/failed_jobs/")
```

Comment:
It's best practice to avoid hard coding as much as possible. Imagine a situation where you are forced to edit the url in the `urls.py` file; you would have to go through the tests (and templates) looking for the url you have just edited.

Long story short, this

`r = self.client.get("/failed_jobs/")`

can be written as

```
from django.urls import reverse

url = reverse("hc-failed_jobs")
r = self.client.get(url)
```

("hc-failed_jobs" is the `name` of the url in the `hc/front/urls.py` file)